src/classes/HMSSDK.tsx

```diff
@@ -135,6 +140,18 @@ export class HMSSDK {
     return getLogger();
   }
 
+  async isPipModeSupported(): Promise<undefined | boolean> {
+    return HMSManager.handlePipActions('isPipModeSupported', null);
+  }
+
+  async enablePipMode(data?: PIPConfig): Promise<undefined | boolean> {
```

Comment:
TODO:= use default `aspect ratio = 16:9` and default `autoenabled  = false` here